manifests/init.pp

```diff
@@ -880,6 +882,17 @@
     validate_re($sshd_config_allowagentforwarding, '^(yes|no)$', "ssh::sshd_config_allowagentforwarding may be either 'yes' or 'no' and is set to <${sshd_config_allowagentforwarding}>.")
   }
 
+  $sshd_acceptenv_defaults_list = [
+    'LANG', 'LC_CTYPE', 'LC_NUMERIC', 'LC_TIME', 'LC_COLLATE',
+    'LC_MONETARY', 'LC_MESSAGES', 'LC_PAPER', 'LC_NAME', 'LC_ADDRESS',
+    'LC_TELEPHONE', 'LC_MEASUREMENT', 'LC_IDENTIFICATION', 'LC_ALL' ]
+
+  if (size($sshd_acceptenv_defaults) == 0 ) {
```

Comment:
If `sshd_acceptenv_custom` is `undef` then we should use `sshd_acceptenv_defaults`, else use `sshd_acceptenv_custom`.

The logic should not assume that the defaults are always wanted and concatenate them together as that goes against the idea of specifying your own custom values if you are always stuck with the defaults.